rubix-tests/src/test/java/com/qubole/rubix/tests/TestNonLocalReadRequestChain.java

```diff
@@ -32,28 +32,45 @@
 import org.apache.hadoop.fs.Path;
 import org.apache.log4j.Level;
 import org.apache.log4j.Logger;
-import org.testng.annotations.AfterMethod;
-import org.testng.annotations.BeforeMethod;
-import org.testng.annotations.Test;
+import org.testng.annotations.*;
```

Comment:
Didnt checkstyle crib about this compaction? 
Easier to read import without compacting them to '*', we should revert this.